app/default/routes.py

```diff
@@ -106,13 +111,20 @@ def tasklist(application_id):
         Config.GET_APPLICATION_ENDPOINT.format(application_id=application_id)
     )
     if not (application_response and application_response["sections"]):
-        return abort(404)
+        return abort(500)
```

Comment:
Can we log something before we abort about what went wrong?

Reply:
Yeh great idea I'll put that in